### PR TITLE
add bengali translation for two brothers campaign/AToTB

### PR DESCRIPTION
**Pull request by knyghtmare**

### CODE REVIEW

**Review by babaissarkar — APPROVED** [review]

File: wesnoth-tb/bn.po [hunk 1]
```diff
@@ -0,0 +1,1293 @@
...
+#. [part]
+#: data/campaigns/Two_Brothers/scenarios/01_Rooting_Out_a_Mage.cfg:57
+msgid ""
+"Then came the day that a dark mage settled in the region and began seeking sacrifices for his summonings.\n"
+"\n"
+"Skeletons and zombies killed cattle and fired fields. <i>“Fear and obey Mordak the Mage!”</i> they cried in fell voices as they did their foul deeds. People vanished from isolated farmsteads. Men and women began to fear the night, and their children even the bright day. But the nearest lord was more than a day’s ride distant, and messengers sent to seek his help did not return."
+msgstr ""
+"তারপরে সেই দিনটি এসেছিল যে একটি অন্ধকার ম্যাজ এই অঞ্চলে বসতি স্থাপন করেছিল এবং তার তলব করার জন্য ত্যাগ চাইতে শুরু করে।\n"
+"\n"
+"কঙ্কাল এবং জম্বিগুলি গবাদি পশু এবং গুলি চালানো ক্ষেতকে হত্যা করেছিল।<i> \"ম্যাজকে ভয় এবং মেনে চলুন!লোকেরা বিচ্ছিন্ন খামার থেকে নিখোঁজ হয়েছিল।পুরুষ এবং মহিলা রাত এবং তাদের সন্তানদের এমনকি উজ্জ্বল দিনকে ভয় করতে শুরু করে।তবে নিকটতম প্রভু এক দিনের রাইডের চেয়ে বেশি দূরের ছিলেন এবং তাঁর সাহায্যের সন্ধানের জন্য প্রেরিত বার্তাবাহকরা ফিরে আসেনি।"
```
Comment (stevecotton) [inline wesnoth-tb/bn.po:112]: `<i>` is italic markup, a bit like HTML - it needs a corresponding `</i>` somewhere

File: wesnoth-tb/bn.po [hunk 2]
```diff
@@ -0,0 +1,1293 @@
...
+
+#. [event]
+#: data/campaigns/Two_Brothers/scenarios/02_The_Chase.cfg:574
+msgid "Katklagad"
+msgstr "ক্যাটল্যাকস"
+
+#. [message]: speaker=unit
+#: data/campaigns/Two_Brothers/scenarios/02_The_Chase.cfg:580
+msgid "Three days ride to the northeast, in a deserted castle. The passwords to the guards are $first_password_$first_password and $second_password_$second_password|."
+msgstr "তিন দিন উত্তর -পূর্ব দিকে যাত্রা করে একটি নির্জন দুর্গে।গার্ডদের পাসওয়ার্ডগুলি হ'ল $ প্রথম_প্যাসওয়ার্ড_ $ প্রথম_প্যাসওয়ার্ড এবং $ সেকেন্ড_প্যাসওয়ার্ড_ $ সেকেন্ড_প্যাসওয়ার্ড |"
```
Comment (stevecotton) [inline wesnoth-tb/bn.po:581]: This one contains WML variables - sometime in the next 24 hours, documentation about how it should be translated will be added to <https://wiki.wesnoth.org/GettextForTranslators>, but it isn't there yet.

**Comment by babaissarkar** [comment]
@stevecotton This textdomain requires cleanup which hasn't been done yet. I plan to do it after the others are a bit stable.

File: wesnoth-tb/bn.po (hunk 1 above)
Comment (babaissarkar) [reply wesnoth-tb/bn.po:112]: Will be cleaned up.

File: wesnoth-tb/bn.po (hunk 2 above)
Comment (babaissarkar) [reply wesnoth-tb/bn.po:581]: Will be cleaned up.

Comment (babaissarkar) [reply wesnoth-tb/bn.po:581]: fixed in ae153a789173dceec199454650b4d83022775829

File: wesnoth-tb/bn.po (hunk 1 above)
Comment (babaissarkar) [reply wesnoth-tb/bn.po:112]: fixed in ae153a789173dceec199454650b4d83022775829.

**Pull request merged**